app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -218,6 +258,21 @@ void testAutomatedClaim_missingAnchor() {
         masResponse.getMessage());
   }
 
+  // Test Case that ensures that exam order *is* callled
+  // The data underlying follows the NEW claim, one relevant condition, not enough information path.
+  @Test
+  void testAutomatedClaim_orderExamNewClaim() {
+    testAutomatedClaimFullPositive("377", false, true);
+  }
```

Comment:
If you give the second parameter false the check for pdf will not be done.  I would reorganize these parameter or modularize testAutomatedClaimFullPositive to check pdf generation as well since ordering exam also puts a pdf in efolder so we want to check that.